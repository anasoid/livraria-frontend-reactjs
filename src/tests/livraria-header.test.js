import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme/build/index';
import LivrariaHeader from '../livraria-header';
import MenuContainer from '../livraria-header/menu-container';

describe('test LivrariaHeader component', () => {
  it('renders without crashing', () => {
    shallow(<LivrariaHeader/>);
    const tree = renderer.create(<LivrariaHeader/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('render MenuContainer', () => {
    const wrapper = shallow(<LivrariaHeader/>);
    expect(wrapper.find(MenuContainer)).toExist();
  });
});