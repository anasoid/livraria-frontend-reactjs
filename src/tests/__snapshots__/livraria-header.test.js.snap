// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test LivrariaHeader component renders without crashing 1`] = `
<header>
  <nav
    className="menu"
  >
    <ul>
      <li>
        <a
          href="/livros"
          id="livroItem"
        >
          Livros
        </a>
      </li>
      <li>
        <a
          href="/autores"
          id="autorItem"
        >
          Autores
        </a>
      </li>
      <li>
        <a
          href="/vendas"
          id="vendaItem"
        >
          Vendas
        </a>
      </li>
      <li>
        <a
          href="/logout"
          id="logoutItem"
        >
          Logout
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
